package.json

```diff
@@ -110,6 +110,7 @@
   },
   "devDependencies": {
     "@types/archiver": "5.1.0",
+    "@types/axios": "^0.14.0",
```

Comment:
See other comments - I think we can avoid this dependency if we write out the commit information when downloading the nuget package.